cfe_internal/enterprise/federation/federation.cf

```diff
@@ -374,7 +361,7 @@ bundle agent federation_manage_files
         template_method => "mustache",
         edit_template => "$(this.promise_dirname)/../../../templates/federated_reporting/config.sh.mustache",
         template_data => mergedata(@(login), @(feeder_username), @(feeder), @(cf_version),
-                                   @(cfengine_enterprise_federation:inventory_refresh_cmd.cmd)),
+                                   parsejson('"inventory_refresh_cmd": ""')),
```

Comment:
That works? Isn't it missing `{` and `}`?

Reply:
bundle agent main
    {
      reports:
        "BEFORE:$(with)" with => string_mustache( "{{%-top-}}", mergedata( @(inventory_refresh_cmd.cmd) ));
        "AFTER: $(with)" with => string_mustache( "{{%-top-}}", mergedata( parsejson('"inventory_refresh_cmd": ""') ));
    }
    bundle agent inventory_refresh_cmd
    {
      vars:
          "cmd" data => parsejson('{"inventory_refresh_cmd":
                                    "$(sys.workdir)/httpd/php/bin/php blah"}');
    }

OUTPUT:

       error: Non-container from parsing JSON expression '": ""'
       error: Non-container from parsing JSON expression '": ""'
    R: BEFORE:{
      "inventory_refresh_cmd": "/home/nickanderson/.cfagent/httpd/php/bin/php blah"
    }
       error: Non-container from parsing JSON expression '": ""'
       error: Non-container from parsing JSON expression '": ""'
    R: AFTER: $(with)
       error: Non-container from parsing JSON expression '": ""'
       error: Non-container from parsing JSON expression '": ""'
       error: Non-container from parsing JSON expression '": ""'
       error: Non-container from parsing JSON expression '": ""'

Reply:
Sigh... I wish `cf-promises cfe_internal/enterprise/federation/federation.cf` would tell me. :disappointed:

Reply:
Fixed and manually tested.